lib/Model/Benchmark.php

```diff
@@ -110,7 +106,7 @@ public function getSuite(): Suite
     }
 
     /**
-     * @return ArrayIterator<Subject>
+     * @return ArrayIterator<array-key, Subject>
```

Comment:
is the `ArrayIterator<Subject>` form not valid? but happy with the more explict form in anycase

Reply:
Yes, it's not valid:

> Method PhpBench\Model\Benchmark::getIterator() return type with generic class ArrayIterator does not specify its types: TKey, TValue